tools/syz-cover/syz-cover.go

```diff
@@ -128,7 +129,10 @@ func main() {
 	if err != nil {
 		tool.Fail(err)
 	}
-	var modules []*vminfo.KernelModule
+	modules, err := backend.DiscoverModules(cfg.SysTarget, cfg.KernelObj, cfg.ModuleObj)
```

Comment:
Let's move 132-140 to some function? Something like initModules().
This main function is quite complex and I experienced a linter cyclo error just a few days ago.

Reply:
That should address your comment